### DISCUSSION TITLE
Image show from folder

### DISCUSSION THREAD

**Discussion by nahidulislam-cse15** [serenity-is/Serenity#6961] [Help]
how can i show image/file in upload folder of one project to other project

**Comment by wezmag** [comment]
In `Startup.cs` file `Configure` method, add following code after `app.UseStaticFiles();`

```
app.UseFileServer(new FileServerOptions
{
    FileProvider = new PhysicalFileProvider("D:\\path\\to\\the\\upload_folder_of_other_project"),
    RequestPath = new PathString("/upload"),
    EnableDirectoryBrowsing = false
});
```

Reference: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-6.0
